packages/core/admin/admin/src/components/AuthenticatedApp/tests/index.test.js

```diff
@@ -0,0 +1,162 @@
+        flex-direction: column;
+        -webkit-box-pack: space-around;
+        -webkit-justify-content: space-around;
+        -ms-flex-pack: space-around;
+        justify-content: space-around;
+        width: 100%;
+        height: 100vh;
+      }
+
+      .c0 > div {
+        margin: auto;
+        width: 50px;
+        height: 50px;
+        border: 6px solid #f3f3f3;
+        border-top: 6px solid #1c91e7;
+        border-radius: 50%;
+        -webkit-animation: fEWCgj 2s linear infinite;
+        animation: fEWCgj 2s linear infinite;
+      }
+
+      <body>
+          <div>
+            <div
+              class="c0"
+            >
+              <div />
+            </div>
+          </div>
+        </body>,
+        "container": <div>
+          <div
+            class="Loader-sc-1xt0x01-0 JXMaR"
+          >
+            <div />
+          </div>
+        </div>,
+        "debug": [Function],
+        "findAllByAltText": [Function],
+        "findAllByDisplayValue": [Function],
+        "findAllByLabelText": [Function],
+        "findAllByPlaceholderText": [Function],
+        "findAllByRole": [Function],
+        "findAllByTestId": [Function],
+        "findAllByText": [Function],
+        "findAllByTitle": [Function],
+        "findByAltText": [Function],
+        "findByDisplayValue": [Function],
+        "findByLabelText": [Function],
+        "findByPlaceholderText": [Function],
+        "findByRole": [Function],
+        "findByTestId": [Function],
+        "findByText": [Function],
+        "findByTitle": [Function],
+        "getAllByAltText": [Function],
+        "getAllByDisplayValue": [Function],
+        "getAllByLabelText": [Function],
+        "getAllByPlaceholderText": [Function],
+        "getAllByRole": [Function],
+        "getAllByTestId": [Function],
+        "getAllByText": [Function],
+        "getAllByTitle": [Function],
+        "getByAltText": [Function],
+        "getByDisplayValue": [Function],
+        "getByLabelText": [Function],
+        "getByPlaceholderText": [Function],
+        "getByRole": [Function],
+        "getByTestId": [Function],
+        "getByText": [Function],
+        "getByTitle": [Function],
+        "queryAllByAltText": [Function],
+        "queryAllByDisplayValue": [Function],
+        "queryAllByLabelText": [Function],
+        "queryAllByPlaceholderText": [Function],
+        "queryAllByRole": [Function],
+        "queryAllByTestId": [Function],
+        "queryAllByText": [Function],
+        "queryAllByTitle": [Function],
+        "queryByAltText": [Function],
+        "queryByDisplayValue": [Function],
+        "queryByLabelText": [Function],
+        "queryByPlaceholderText": [Function],
+        "queryByRole": [Function],
+        "queryByTestId": [Function],
+        "queryByText": [Function],
+        "queryByTitle": [Function],
+        "rerender": [Function],
+        "unmount": [Function],
+      }
+    `);
+  });
+
+  it('should not fetch the latest release', () => {
```

Comment:
Could you explain why? or "when" it shouldn't fetch the latest release?

Reply:
There is an option to disable this external api call.

Reply:
can you make it explicit in the title?